src/components/filters/date-range-picker/date-range-picker.styles.js

```diff
@@ -24,21 +24,28 @@ export const useStyles = makeStyles((theme) => ({
     }
   },
   date: {
-    width: 250,
+    width: 290,
     '& span': { color: '#828282 !important' },
     '& a.rs-btn.rs-btn-default.rs-picker-toggle.active.rs-btn-md': {
       borderColor: 'black !important'
     },
     '& toggle.active': { borderColor: 'red !important' }
   },
   menu: {
+    width: 290,
     '& button': {
       backgroundColor: '#3f51b5 !important',
       color: '#ffffff !important'
     },
     '& .rs-calendar-table-cell-selected .rs-calendar-table-cell-content': {
       backgroundColor: '#3f51b5 !important',
       borderColor: '#3f51b5 !important'
+    },
+    '& .rs-picker-toolbar': {
+      maxWidth: '290px !important'
```

Comment:
Please check this css file and delete all "!important"s or replace them with similar code :3

Reply:
Some importants are necessary